### DISCUSSION TITLE
补充几个群晖的qc连接需要的域名排除

### DISCUSSION THREAD

**Discussion by Lovest20018** [kkkgo/PaoPaoDNS#75] [经验分享]
默认qc可能会因为fakeip问题的连接不上，
把下面域名添加到force_cn_list.txt

domain:checkip.synology.com
domain:checkipv6.synology.com
domain:checkport.synology.com
domain:ddns.synology.com
domain:account.synology.com
domain:quickconnect.cn

**Comment by kkkgo** [comment]
这种情况可以加入`force_nocn_list.txt`。`force_cn_list.txt`对于境外域名不一定能递归成功。

**Comment by kkkgo** [comment]
群晖的qc应该是有中国大陆区域的吧，好像是登录国区账户就是分配国区的服务器，上面的域名应该不会影响到QC。
看起来更像是影响DDNS。
顺便推荐下DDNS的项目https://github.com/kkkgo/UE-DDNS

**Comment by Lovest20018** [comment]
qc是蛮早就有国区的服务器了，上面最后一个域名应该是qc国内的域名，一般都是以账号名+.quickconnect.cn来获得中继，
没添加的时候一直连不上qc，加了中继就正常了。

**Reply by kkkgo** [reply]
quickconnect.cn按道理是国内域名，本身就不会被FAKEIP。感觉是`account.synology.com`影响吧
